src/nuget/Microsoft.DotNet.Build.Tasks.Feed.nuspec

```diff
@@ -20,11 +20,11 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="Microsoft.DotNet.Build.Tasks.Feed\Microsoft.DotNet.Build.Tasks.Feed.dll" target="lib" />
-    <file src="Microsoft.DotNet.Build.Tasks.Feed.net45\*.dll" target="lib" Exclude="Microsoft.DotNet.Build.Tasks.Feed\Microsoft.DotNet.Build.Tasks.Feed.dll"/>
-    <file src="Microsoft.DotNet.Build.Tasks.Feed.net45\*.dll" target="lib\desktop" />
-    <file src="Microsoft.DotNet.Build.Tasks.Feed\PackageFiles\**\*.*" target="lib" />
-    <file src="Microsoft.DotNet.Build.Tasks.Feed\PackageFiles\**\*.*" target="lib\desktop" />
+    <file src="Microsoft.DotNet.Build.Tasks.Feed\Microsoft.DotNet.Build.Tasks.Feed.dll" target="lib\netstandard2.0" />
```

Comment:
why not netstandard1.5 which is what you are targeting?

Reply:
Is there a minimum tfm that we need to target? What tfm does buildtools usually support?

Reply:
I think there are a number of init-tools.msbuild projects that are restoring for netcoreapp1.0 so in order for this to be picked up it will need to be at least netstandard1.6 or less. I suggest you stick with netstandard1.5 which is what you are targeting in the build.

Reply:
updated to match 1.5

Reply:
LGTM